tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -112,6 +112,40 @@ def test_ebs_single_empty(scheduler, pcluster_config_reader, clusters_factory):
     _test_ebs_correctly_shared(remote_command_executor, mount_dir, scheduler_commands)
 
 
+@pytest.mark.dimensions("ap-northeast-2", "c5.xlarge", "alinux2", "sge")
+@pytest.mark.usefixtures("os", "instance")
+def test_ebs_existing(
+    request, vpc_stacks, region, scheduler, pcluster_config_reader, clusters_factory, snapshots_factory
+):
+    logging.info("Testing ebs existing")
+    existing_mount_dir = "existing_mount_dir"
+
+    logging.info("Creating volume id")
```

Comment:
minor, terminology: you're creating a volume, and that volume happens to have an ID. i would say 'Creating volume' instead

Reply:
Right! Done